lib/pipeline.py

```diff
@@ -165,21 +157,10 @@ def convert_vcf(self):
             'sample_name': self.sample_name,
             'pass_only': self.pass_only,
         }
-        for attribute in [
-            'gene_expn_file',
-            'transcript_expn_file',
-            'normal_snvs_coverage_file',
-            'normal_indels_coverage_file',
-            'tdna_snvs_coverage_file',
-            'tdna_indels_coverage_file',
-            'trna_snvs_coverage_file',
-            'trna_indels_coverage_file',
-            'normal_sample_name',
-        ]:
-            if getattr(self, attribute):
-                convert_params[attribute] = getattr(self, attribute)
-            else:
-                convert_params[attribute] = None
+        if self.normal_sample_name is not None:
+            convert_params['normal_sample_name'] = self.normal_sample_name
+        else:
+            convert_params['normal_sample_name'] = None
```

Comment:
Could this conditional be eliminated entirely? Looks like this is set to `self.normal_sample_name` either way.

Reply:
This is actually wrong the way I wrote this. The else clause isn't supposed to be in there. I removed it.